src/third-party/doc/2.76.0/pdf.worker.js

```diff
@@ -35682,13 +35724,15 @@ var Font = function FontClosure() {
     },
 
     exportData: function Font_exportData() {
-      var data = {};
-
-      for (var i in this) {
-        if (this.hasOwnProperty(i)) {
-          data[i] = this[i];
+      var data = Object.create(null);
+      var that = this, value;
```

Comment:
Super minor, but in production code we would want either one `var` per variable or to use one `var` with a comma separated list of variables. We have both formats here.